packages/statemanager/CHANGELOG.md

```diff
@@ -6,3 +6,73 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+
+So, enjoy the releases (this is a first round of Beta releases, with final releases following a couple of weeks after if things go well)! 🎉
+
+The EthereumJS Team
+
+### New Package
+
+The `StateManager` has been extracted from the `VM` and is now a separate package, see PR [#1817](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1817). The new package can be installed separately with:
+
+```shell
+npm i @ethereumjs/statemanager
+```
+
+The `@ethereumjs/vm` package still has this package added as a dependency and it is automatically integrated. The `StateManager` provides a high-level interface to an underlying state storage solution. This is classically a `Trie` (in our case: an `@ethereumjs/trie`) instance, but can also be something else, e.g. a plain database, an underlying RPC connection or a Verkle Tree in the future.
+
+The extraction of this module allows to easier customize a `StateManager` and provide or use own implementations in the future. It now also gets possible to use the `StateManager` standalone for high-level state access in a non-VM context.
+
+A `StateManager` must adhere to a predefined interface `StateManager` and implement a certain set of state access methods like `getAccount()`, `putContractCode()`,... Such an implementation is then guaranteed to work e.g. in the `@ethereumjs/vm` implementation.
+
+### StateManager Refactoring
+
+Along with the package extraction parts of the old `StateManager` has also been reworked. So if you are building on the old `StateManager` class/interface it is likely not enough to just change on the import statement but do some adjustments to get things working. Here is a summary of the changes.
+
+Methods added:
+
+- `flush()`
+
+Methods removed:
+
+- `touchAccount()` (EVM-specific, remained in `EVMStateAccess` interface in EVM)
+- All methods from `EIP2929StateManager` (removed as separate interface) (EVM-specific, remained in `EVMStateAccess` interface in EVM)
+- `getOriginalContractStorage()` (EVM-specific, remained in `EVMStateAccess` interface in EVM)
+- `hasGenesisState()` (removed)
+- `generateGenesis()` (removed)
+- `generateCanonicalGenesis()` (EVM-specific, remained in `EVMStateAccess` interface in EVM)
+- `cleanupTouchedAccounts()` (EVM-specific, remained in `EVMStateAccess` interface in EVM)
+- `clearOriginalStorageCache()` (EVM-specific, remained in `EVMStateAccess` interface in EVM)
+
+Other Changes:
+
+- New partial parent interface `StateAccess` with just the access focused functionality
+
+So overall the `StateManager` interface got a lot leaner requesting fewer methods to be implemented which should make an implementation and/or adoption a lot easier.
+
+The `StateManager` package ships with a Trie-based `StateManager` implementation extending from a `BaseStateManager` which might be a suitable starting point for own implementations. This will very much depend on the specific needs though.
```

Comment:
```suggestion
The `StateManager` package ships with a Trie-based `StateManager` implementation extending from a `BaseStateManager` which might be a suitable starting point for your own implementations. This will very much depend on the specific needs though.
```